radio/src/opentx.cpp

```diff
@@ -2488,16 +2488,14 @@ void opentxInit(OPENTX_INIT_ARGS)
   rtcInit();    // RTC must be initialized before rambackupRestore() is called
 #endif
 
-#if !defined(EEPROM)
-  if (!UNEXPECTED_SHUTDOWN()) {
-    sdInit();
+  if (UNEXPECTED_SHUTDOWN()) {
+    unexpectedShutdown = 1;
   }
+  else {
+#if defined(SDCARD)
+    sdInit();
 #endif
-
-#if defined(COLORLCD)
-  topbar = new Topbar(&g_model.topbarData);
-  luaInit();
-#endif
+  }
 
 #if defined(RAMBACKUP)
```

Comment:
I don't remember what happens if lua is not initialized here, in case the screens have lua widgets inside